packages/cfd/src/Components/__tests__/cfd-mt5-demo-account-display.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import CFDMT5DemoAccountDisplay from '../cfd-mt5-demo-account-display';
+import { CFD_PLATFORMS } from '@deriv/shared';
+
+//mock cfd-account-card
+jest.mock('../cfd-account-card', () => ({
+    CFDAccountCard: () => <div>CFDAccountCard</div>,
+}));
+
+describe('CFDMT5DemoAccountDisplay', () => {
+    const mock_standpoint = {
+        financial_company: 'financial_company',
+        gaming_company: 'gaming_company',
+        iom: false,
+        malta: false,
+        maltainvest: false,
+        svg: true,
+    };
+
+    const mock_props: React.ComponentProps<typeof CFDMT5DemoAccountDisplay> = {
+        is_eu: false,
+        is_eu_country: false,
+        has_maltainvest_account: false,
+        has_cfd_account_error: false,
+        openAccountNeededModal: () => jest.fn(),
+        standpoint: mock_standpoint,
+        is_loading: false,
+        is_logged_in: true,
+        isSyntheticCardVisible: jest.fn(() => true),
+        isFinancialCardVisible: jest.fn().mockReturnValue(false),
+        onSelectAccount: jest.fn(),
+        openAccountTransfer: jest.fn(),
+        platform: CFD_PLATFORMS.MT5,
+        current_list: {},
+        openPasswordManager: jest.fn().mockReturnValue(true),
+        residence: 'country',
+        landing_companies: {},
+        toggleMT5TradeModal: jest.fn(),
+        show_eu_related_content: false,
+    };
+    it('should render a loading component when is_loading is true', () => {
+        render(<CFDMT5DemoAccountDisplay {...mock_props} is_loading={true} />);
+        expect(screen.getByTestId('dt_barspinner')).toBeInTheDocument();
+    });
+
+    it('should render a loading component when is_loading is false', () => {
+        render(<CFDMT5DemoAccountDisplay {...mock_props} is_loading={false} />);
```

Comment:
Isn't `is_loading={false} `declared by default? In line 28

Reply:
Totally slipped out that part
Done